app/Http/Controllers/Equipment/RepairsController.php

```diff
@@ -63,14 +64,28 @@ public function store(RepairRequest $request)
             'status'      => Breakage::STATUS_REPORTED,
             'user_id'     => $request->user()->id,
             'closed'      => false,
-        ]);
+        ]); 
+        // Save the images
+        if ($request->hasFile('images')) {
+            $images = $request->file('images');
+            $i = 0;
+            foreach ($images as $image) {
```

Comment:
You could probably use an indexed foreach here, to save having to explicitly declare `$i`:

```php
foreach ($images as $i => $image) {
```

Reply:
Dropped the `position_id` so it won't be necessary to keep track of the index